src/main/java/com/exasol/adapter/dialects/AbstractSqlDialect.java

```diff
@@ -14,7 +14,11 @@
 import com.exasol.adapter.jdbc.ConnectionFactory;
 import com.exasol.adapter.jdbc.RemoteMetadataReader;
 import com.exasol.adapter.metadata.SchemaMetadata;
-import com.exasol.adapter.sql.*;
+import com.exasol.adapter.sql.AggregateFunction;
+import com.exasol.adapter.sql.ScalarFunction;
+import com.exasol.adapter.sql.SqlNodeVisitor;
+import com.exasol.adapter.sql.SqlStatement;
```

Comment:
I think you have a problem with the formatter here. This should compress to a single import as it was before

Reply:
I formated again using the latest configuration. Seems like it is currently configured like that